data/map.txt

```diff
@@ -30817,6 +31116,14 @@ planet "Sopi Lefarkata"
 	bribe 0
 	security 0
 
+planet "Spec Inci"
+	attributes "hai quarg" quarg station
+	landscape land/station19
+	description `Although you have seen a lot of spacestations you are always impressed by the Quarg ringworlds, constructions that replace a whole planet.`
```

Comment:
This description is also too short, and doesn't add any new information.

Reply:
ok